invenio_pidstore/__init__.py

```diff
@@ -187,7 +187,17 @@
 Invenio-Records installed in order for this example to work):
 
 >>> from invenio_pidstore.resolver import Resolver
->>> from invenio_records.api import Record
+>>> try:
+...     from invenio_records.api import Record
+... except ImportError:
+...     class Record(dict):
```

Comment:
I don't think the basic usage documentation should expose this level of complexity. Perhaps it's possible to mock  ``from invenio_records.api import Record`` via ``.. testcode::`` (with hide option) or ``.. testsetup::``